src/util/validators/string-validator.ts

```diff
@@ -1,12 +1,15 @@
 interface StringValidationRules {
   nonEmpty?: boolean,
   minLength?: number,
-  maxLength?: number
+  maxLength?: number,
+  regexp?: RegExp
 }
 
 export const validateString = (string: string, rules: StringValidationRules) => {
+  if (!string) return false;
```

Comment:
This will catch empty strings, since `boolean('') === false`

Reply:
changed to `string === undefined || string === null`

Reply:
Could also be `string == null`